src/codeu/chat/server/View.java

```diff
@@ -78,6 +78,81 @@ public View(Model model) {
   @Override
   public Message findMessage(Uuid id) { return model.messageById().first(id); }
 
+  @Override
+  public Collection<String> userStatusUpdate(String name, Uuid owner) {
+
+    Collection<ConversationPayload> allConversations = all(model.conversationPayloadById());
+    Collection<String> contributions = new ArrayList<String>();
+
+    final User foundOwner = model.userById().first(owner);
+    final User foundUser = model.userByText().first(name);
+
+    if(foundUser != null) {
+      if(foundOwner.UserSet.contains(foundUser.id)) {
```

Comment:
Oh, whoops. Meant to say foundUser. I'm pretty sure that foundOwner can't be null in this case because the UUID provided as the argument is the UUID of the user that is currently signed in. It's impossible to sign in as a user if they're not in Model (and if they are, they have a UUID.